index.html

```diff
@@ -196,40 +207,54 @@ <h2>About</h2>
 <!-- foooter -->
 <footer class="contact_section">
     <div class="mobile_logo"></div>
+
+    <div class="footer_div">
+
+
     
-      <h4 class="title-contact text_header_font">Get Started</h4>
-
-      <p class="section-desc text_body_font">
-        Hello!I am a software developer! I can 
-        help you build a product, feature or website.
-        Take a look of my works. If you like what you see 
-        and have a project you need coded, don't hesitate 
-        and contact me.
-      </p>
-      <form action="https://formspree.io/f/mzbovllg" method="post">
-          <input type="text" name="FullName" placeholder="Enter Your Name" maxlength="30" required>
-          <input type="email" name="Email" placeholder="Enter Your Email" required>
-          <textarea name="Message" placeholder="Enter Your Message"  maxlength="500" required></textarea>
-          <button class="" type="submit" name="submit-btn">Start collaboration</button>
-      </form>
-
-      <div class="bottom_row">
-        <a class="text-button text_body_font resume_color">
-            <p>Get my Resume</p>
-            <div class="arrow_upload"></div>
-        </a>
-
-        <div class="divicon">
-            <ul class="sociallist">
-           <li class="sociallist_link"><a href="#"><img src="img/twitter.png" alt="twitter"></a></li>
-           <li class="sociallist_link" ><a href="#"><img src="img/linkdin.png" alt="linkdin"></a></li>
-           <li class="sociallist_link" ><a href="#"><img src="img/medium.png" alt="medium"></a></li>
-           <li class="sociallist_link"><a href="#"><img src="img/github.png" alt="github"></a></li>
-           
-            </ul>
+    
+        <div class="contact_div">

```

Comment:
- [ ] In the `Contact me` section, Please add the background image as in the original website.
- Your website 
![Screenshot from 2022-03-27 03-31-45](https://user-images.githubusercontent.com/26767383/160258636-5332df09-f49f-4c59-a70c-d3369b87fa77.png)
- Figma website 
![Screenshot from 2022-03-27 03-32-39](https://user-images.githubusercontent.com/26767383/160258644-a67f10d6-a968-42e0-b923-57a35a715f42.png)